daml-lf/interpreter/BUILD.bazel

```diff
@@ -10,6 +10,10 @@ load(
 )
 load("@io_bazel_rules_scala//scala:scala.bzl", "scala_repl")
 
+lf_scalacopts_stricter = lf_scalacopts + [
```

Comment:
maybe move this next to the definition of `lf_scalacopts`?

Reply:
ok, I'll look into doing this

Reply:
did it